fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreTestBase.java

```diff
@@ -146,34 +175,14 @@ protected RecordLayerPropertyStorage.Builder addDefaultProps(RecordLayerProperty
         return props;
     }
 
-    @BeforeEach
-    public void clearAndInitialize() {
-        getFDB();
-        clear();
-
-        // Reset these indexes added and last modified versions, which can be updated during tests.
-        // For example, adding the indexes to a RecordMetaDataBuilder can update these fields
-        COUNT_INDEX.setAddedVersion(0);
-        COUNT_INDEX.setLastModifiedVersion(0);
-        COUNT_UPDATES_INDEX.setAddedVersion(0);
-        COUNT_UPDATES_INDEX.setLastModifiedVersion(0);
-    }
-
-    protected void clear() {
-        fdb.run(timer, null, context -> {
-            path.deleteAllData(context);
-            return null;
-        });
-    }
-
-    @AfterEach
-    public void checkForOpenContexts() {
-        int count = fdb.warnAndCloseOldTrackedOpenContexts(0);
-        assertEquals(0, count, "should not have left any contexts open");
+    @Nonnull
+    public static Index globalCountIndex() {
+        return new Index(COUNT_INDEX_NAME, new GroupingKeyExpression(EmptyKeyExpression.EMPTY, 0), IndexTypes.COUNT);
```

Comment:
The static `Index` objects can cause trouble because they are actually semi-stateful. In particular, the `RecordMetaDataBuilder` can update the last modified and and added versions (which is why the `clearAndInitialize` method used to reset them to zero with every test). Now, we just create a new instance rather than using static ones.